components/rating/index.css

```
@@ -40,9 +81,11 @@ governing permissions and limitations under the License.
   display: inline-flex;
   flex: 0 0 auto;
 
-  inline-size: calc(var(--spectrum-rating-icon-width) * var(--spectrum-rating-icon-count));
+  inline-size: calc((var(--mod-rating-icon-width, var(--spectrum-rating-icon-width)) + var(--mod-rating-icon-spacing, var(--spectrum-rating-icon-spacing))) * var(--mod-rating-icon-count, var(--spectrum-rating-icon-count)));
 
-  border-radius: var(--spectrum-rating-border-radius);
+  border-radius: var(--mod-rating-border-radius, var(--spectrum-rating-border-radius));
+
+  /*size: var(--mod-rating-workflow-icon-size, var(--spectrum-rating-workflow-icon-size));*/
 
```

A really nice feature of flexbox is that we can use `gap` to provide nice spacing between each of the flex-children. Please add the line below:

```suggestion
gap: var(--mod-rating-icon-spacing, var(--spectrum-rating-icon-spacing));
```